backend/metadata_writer/src/metadata_writer.py

```diff
@@ -0,0 +1,333 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import hashlib
+import os
+import sys
+import kubernetes
+from time import sleep
+
+from metadata_helpers import *
+
+
+namespace_to_watch = os.environ.get('NAMESPACE_TO_WATCH', 'default')
+
+
+try:
+    kubernetes.config.load_incluster_config()
+except Exception as e:
+    print(e)
+    try:
+        kubernetes.config.load_kube_config()
+    except Exception as e:
+        print(e)
+
```

Comment:
I'll fix this. Thanks.

(Technically, this is not a problem (since it's not required to call either `load_incluster_config` or `load_kube_config` and the error should not happen and the error would have been detected anyways when calling `k8s_api.list_namespaced_pod`). But removing these conditionals will improve readability.).

Reply:
I've reworked this code block.